spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample/README.adoc

```diff
@@ -8,25 +8,25 @@ link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-data-spanner[Sprin
 
 1. Make sure that you have the Cloud SDK configured by following https://cloud.google.com/sdk/docs/[these instructions].
 
-2. Create a new Spanner instanced named "spring-demo" with a database called "trades"
-and a table also named "trades".
-The columns should be created as follows:
-+
+2. Setup the database
+
+a.) Create a new Spanner instance named "spring-demo". For testing, choose the cheapest region possible for you and only 1 region instance should suffice.
+b.) Create with a database called "trades"
+c.) Create a table, also named "trades". You can define the table via the following DDL
+
 ```
-action , STRING
-price , FLOAT64
-shares , FLOAT64
-symbol , STRING
-trader_id , STRING
+CREATE TABLE trades (
+  action STRING(15),
+  price FLOAT64,
+  shares FLOAT64,
+  symbol STRING(10),
+  trader_id STRING(128)
+) PRIMARY KEY (action, symbol)
 ```
 
-The columns `action` and `symbol` should be set as the composite primary key in that order.
-
 3. Place your project ID in the `spring.cloud.gcp.spanner.projectId` setting in `application.properties`
 if you are not providing the project ID from a JSON key file or the SDK.
```

Comment:
We're not getting the project from the credentials, so this sentence is not 100% accurate.

Seems like you need an extra bullet point to mention credentials.